.gitignore

```diff
@@ -56,3 +56,5 @@ pytensor-venv/
 testing-report.html
 coverage.xml
 .coverage.*
+pics
+*.ipynb
```

Comment:
Not sure about this. `pics` looks very specific, and `.ipynb` too restrictive

Reply:
Ah this one is the file on my local machine. `pics` is where I store the graph pictures above. `.ipynb` is because I do the testing inside Jupyter Notebook. I don't want them to get merged so I add these. Will delete before merging.